lib/Feature/FeatureModelParser.cpp

```diff
@@ -93,14 +81,51 @@ bool FeatureModelXmlParser::parseConfigurationOption(xmlNode *Node,
       }
     }
   }
+  std::optional<std::vector<FeatureSourceRange>> Locations =
+      SourceRanges.empty() ? std::nullopt : std::make_optional(SourceRanges);
```

Comment:
```suggestion
      SourceRanges.empty() ? std::nullopt : std::make_optional(std::move(SourceRanges));
```
Looks like this is no longer used afterwards.